src/components/Button/Button.spec.js

```diff
@@ -5,12 +5,12 @@ import '@testing-library/jest-dom';
 import {render, screen} from '@testing-library/react';
```

Comment:
As your button is only a styled-component you do not need a test and story for it - please delete these files. 🙂